flycheck.el

```diff
@@ -5782,7 +5929,8 @@ See URL `http://clang.llvm.org/'."
               (or (flycheck-error-message err) "no message")))
       (flycheck-fold-include-levels errors "In file included from")))
   :modes (c-mode c++-mode)
-  :next-checkers ((warning . c/c++-cppcheck)))
+  :next-checkers ((warning . c/c++-cppcheck))
+  :conflicts-with c/c++-gcc)
```

Comment:
@cpitclaudel I do not think that running Clang and GCC on the same file makes sense.  I figure that you'd get a lot of duplicates.  Each one may have their unique warnings but the vast majority is the same isn't it?

And in any case you wouldn't build software with both would you? Typically you've got just one target compiler, don't you?